src/tools/cephfs/tox.ini

```diff
@@ -0,0 +1,8 @@
+[tox]
+envlist = py3 flake8
+skipsdist = true
```

Comment:
why is this set? it will not install cephfs-shell

Reply:
cephfs-shell is not available as package on pypi

Reply:
that is not what `skipsdist = true` means. From the docs:

```
Flag indicating to perform the packaging operation or not. Set it to true when using tox for an application, instead of a library.
```
That means for cephfs-shell you *want* to have tox install via `setup.py` so that it runs against it, it doesn't mean "install cephfs-shell from pypi"